### ISSUE TITLE
one bug, one suggestion

### ISSUE THREAD

**Issue by camilodelvasto** [chipcullen/fontcombinator-legacy#20]
Just found this nice tool and would love to contribute. For now I've found one issue:

The fonts dropdown list doesn't show the font name unless you are hovering it. Actually, it's just inheriting the color from the form element (which is white). So adding:
`.chosen-container { color: black; }` solves it. 

On the other hand, it might be nice to have different colors or shades of black on the first screen, to let the user know there are three editable elements (for example, making the subhead gray by default).

Thanks for the tool!

**Comment by mkrdip** [comment]
Yes, for both dropdown items have white color as like background, so they don't show. Secreenshot: 

![image](https://cloud.githubusercontent.com/assets/3819579/12787437/a18ce96c-cabf-11e5-8ec2-ab6d80ea7278.png)

To be more specific, the following CSS codes resolves it:
`.chosen-container .chosen-drop { color: #000; }`
